src/pauses.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Returns the position and types of pauses (vishraams) in a gurbani line
+ *
+ * @since 1.0.0
+ * @param {string} words The string from which to get the pause positions
+ * @param {string} large the symbol to use for a large pause (vishraam)
+ * @param {string} medium the symbol to use for a medium pause (jamki)
+ * @param {string} small the symbol to use for a small pause (thamki)
+ * @returns {JSON} Returns the position and type for each pause
+ * @example
+ *
+ * vishraams('ikv. sicAwrw hoeIAY; ikv. kUVY qutY pwil ]');
+ * // => '[{"p": 0, "t": "t"}, {"p": 2, "t": "v"}, {"p": 3, "t": "t"}]'
+ */
+
+
+function pauses(words = '', large = ';', medium = ',', small = '.') {
```

Comment:
as mentioned in https://github.com/KhalisFoundation/anvaad-js/pull/75#discussion_r366112924 1 param would be preferred so order becomes a non-issue.

Reply:
this doesn't seem to work with our implementation of default params, is there a different syntax for that?

Reply:
https://www.sitepoint.com/es6-default-parameters/ 

basically something like.
```
function pauses (words = '', {
  large = ';',
  medium = ',',
  small = '.',
} = {}) {
```

alternatively we can overload the function as a follow up PR, that way several definitions can exist and a user can call the function any way they like. internally they can all call this.